### PR TITLE
Lib: Add an index file for all exports

### PR DESCRIPTION
**Pull request by mia-pi-git**
Discussed with @Zarel. Ideally, this can be a standalone module next.

### CODE REVIEW

File: lib/utils.ts [hunk 1]
```diff
@@ -199,155 +200,157 @@ export const Utils = new class {
...
+}
+
+export function waitUntil(time: number): Promise<void> {
+	return new Promise(resolve => {
+		setTimeout(() => resolve(), time - Date.now());
+	});
+}
+
+// backwards compatibility
+export const Utils = module.exports;
```
Comment (Zarel) [inline lib/utils.ts:356]: I'd prefer not to mix CommonJS and ES modules. It's probably better to just name every function again here.

Comment (mia-pi-git) [reply lib/utils.ts:356]: Alright.

**Review by KrisXV — APPROVED** [review]

**Comment by Zarel** [comment]
tbh I didn't expect you to still import everything individually. It doesn't seem much different from before. :s

**Comment by Zarel** [comment]
Writing down what we discussed in DMs:

I do still want to, at some point, release a generalized library called `libio`, which exports a variable named `IO`, with most things in `lib` under it.

But I think this is pretty fine for now. I think we should think harder of how we'd want to package `libio` if we wanted to go much further along that route.

**Pull request merged**